masterlist.yaml

```
@@ -8586,41 +8586,71 @@ plugins:
+        subs: [ 'Cosmetic Vampire Overhaul' ]
+        condition: 'active("CosmeticVampireOverhaul.esp") and not active("MLU - CosmeticVampireOverhaul.esp") and not active("MLU Patches Merged.esp")'
+      - <<: *patchProvided
+        subs: [ 'utting Room Floor' ]
+        condition: 'active("Cutting Room Floor.esp") and not active("MLU - CRF.esp") and not active("MLU Patches Merged.esp")'
       - <<: *patchProvided
         subs: [ 'Immersive Armors' ]
         condition: 'active("Hothtrooper44_ArmorCompilation.esp") and not active("MLU - Immersive Armors.esp") and not active("MLU Patches Merged.esp")'
       - <<: *patchProvided
-        subs: [ 'Improved closefaced helmets' ]
+        subs: [ 'Improved Closefaced Helmets' ]
         condition: 'active("imp_helm_legend.esp") and not active("OMEGA MLU Improved Closefaced Helmets.esp") and not active("MLU - Improved Closefaced Helmets.esp") and not active("MLU Patches Merged.esp")'
+      - <<: *patchProvided
+        subs: [ 'Improved Daedric Artifacts' ]
+        condition: 'active("ImprovedDaedricArtifacts.esp") and not active("MLU - ImprovedDaedricArtifacts.esp") and not active("MLU Patches Merged.esp")'
+      - <<: *patchProvided
+        subs: [ 'INIGO' ]
+        condition: 'active("Inigo.esp") and not active("MLU - ImprovedDaedricArtifacts.esp") and not active("MLU Patches Merged.esp")'
       - <<: *patchProvided
         subs: [ 'InsanitySorrow Weapons Pack' ]
         condition: 'active("InsanitySorrow Weapons Pack.esp") and not active("MLU - InsanitySorrow Weapons Pack.esp") and not active("MLU Patches Merged.esp")'
+      - <<: *patchProvided
+        subs: [ 'JK''s The Drunken Huntsman' ]
+        condition: 'active("JKs The Drunken Huntsman.esp") and not active("MLU - JK''s The Drunken Hunstman.esp") and not active("MLU Patches Merged.esp")'
       - <<: *patchProvided
         subs: [ 'Lore Weapon Expansion' ]
-        condition: 'active("Lore Weapon Expansion.esp") and not active("MLU - Lore Weapon Expansion.esp") and not active("MLU Patches Merged.esp")'
+        condition: 'active("Lore Weapon Expansion.esp") and not active("LegacyoftheDragonborn.esm") and not active("MLU - Lore Weapon Expansion.esp") and not active("MLU Patches Merged.esp")'
+      - <<: *patchProvided
+        subs: [ 'Lore Weapon Expansion and Legacy of the Dragonborn' ]
+        condition: 'active("Lore Weapon Expansion.esp") and active("LegacyoftheDragonborn.esm") and not active("MLU - Lore Weapon Expansion - LOTD Version.esp") and not active("MLU Patches Merged.esp")'
       - <<: *patchProvided
         subs: [ 'Mainland Stalhrim' ]
         condition: 'active("MainlandStalhrim.esp") and not active("MLU - Mainland Stalhrim.esp") and not active("MLU Patches Merged.esp")'
+      - <<: *patchProvided
+        subs: [ 'Simple Magic Rebalance' ]
+        condition: 'active("Simple Magic Rebalance.esp") and not checksum("Simple Magic Rebalance.esp", 4F3CB180) and not active("MLU Patches Merged.esp")'
```

When using the `checksum` function, it's best to look for specific versions of a file rather than everything but a specific version. Doing so will result in a lot of false positives (e.g. if someone makes an edit to an ESP like adding an ESL flag). It'll cause a lot less confusion if people just don't see the message rather than install the patch, make a small edit, then get the message again as a result.

```suggestion
        condition: 'active("Simple Magic Rebalance.esp") and (checksum("Simple Magic Rebalance.esp", D55C09E9) or checksum("Simple Magic Rebalance.esp", 971ADB2B)) and not active("MLU Patches Merged.esp")'
```